.github/workflows/thank-contributor.yml

```
@@ -0,0 +1,39 @@
+name: "Thank contributor for contribution"
+on:
+  pull_request:
+    types:
+      - closed
+jobs:
+  Congratulate:
+    if: github.event.pull_request.merged == true && github.event.pull_request.user.login != 'dependabot[bot]'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Sleep for 120 seconds
+        run: sleep 120s
+        shell: bash
+      - name: Thank contributor for contribution
+        env:
+          ISSUES_BOT_TOKEN: ${{ secrets.ISSUES_BOT_TOKEN }}
+        run: |
+          username="${{ github.event.pull_request.user.login }}"
+          repo="${{ github.repository }}"
+          if [[ "$username" =~ [[:punct:]] ]]; then
+            message="Hi @$username. :wave:\n\nYour PR has been approved and merged. :tada:\n\nThank you for your contribution to the starter.dev GitHub showcases repo. :heart:\n\nKeep up the great work!"
+          else
+            response=$(curl -s -o /dev/null -w "%{http_code}" -H "Authorization: Bearer $ISSUES_BOT_TOKEN" "https://api.github.com/repos/$repo/collaborators/$username")
+            if [ "$response" = "204" ]; then
+              message="Hi @$username. :wave:\n\nYour PR has been approved and merged. :tada:\n\nThank you for your continued contributions to the starter.dev GitHub showcases repo. :heart:\n\nKeep up the great work!"
+            elif [ "$response" = "404" ]; then
+              message="Hi @$username. :wave:\n\nCongrats on your first PR being approved and merged into the starter.dev GitHub showcases repo! :tada:\n\nThank you for taking the time to contribute to our project. :heart:\n\nWe look forward to your next contribution. :rocket:"
```

Should this also be the project name instead of "project"? 
```suggestion
              message="Hi @$username. :wave:\n\nCongrats on your first PR being approved and merged into the starter.dev GitHub showcases repo! :tada:\n\nThank you for taking the time to contribute to the starter.dev GitHub showcases repo. :heart:\n\nWe look forward to your next contribution. :rocket:"
```